src/UglyToad.PdfPig.Tests/Geometry/PdfLineTests.cs

```diff
@@ -21,11 +24,315 @@ public void OriginIsZero()
         [Fact]
         public void Length()
         {
-            var line = new PdfLine(2, 1, 6, 4);
-            Assert.Equal(5d, line.Length);
+            Assert.Equal(578.0238169967487, new PdfLine(772.7935893651529, 299.87394515637465, 233.21554238153897, 92.58896203207878).Length, 6);
```

Comment:
For most of these Geometry tests it'd be better to move all this repeated test data to a `Theory` using `MemberData`: http://hamidmosalla.com/2017/02/25/xunit-theory-working-with-inlinedata-memberdata-classdata/

Reply:
Okay good point 😄 I did it it this way because the tests were generated via code. Will look at it!